docs/development/environment-setup/docker.md

```diff
@@ -1,76 +1,108 @@
 ---
 path: "/docs/installation-guide-docker"
-title: "Docker Setup and Debugging Guide"
+title: "Docker Setup"
 ---
 
-### Development environment
+### Pre-requisites
+Before proceeding, make sure you have the latest version of `docker` and `docker-compose` installed.
 
-After cloning the repo and installing docker on your machine, run the following command from the root directory of the project.
+As of now[at the time of writing this doc], we recommend 
 
-```bash
-cp .env.example .env
-```
+    ```bash
+    $ docker --version
+    Docker version 19.03.3, build a872fc2f86
+    $ docker-compose --version
+    docker-compose version 1.25.3, build d4d1b42b
+    ```
 
-Make changes to the `.env` file as required [Optional]. If you want to set the password for redis when you run
-docker-compose, set any string value to the environment variable `REDIS_PASSWORD` in the `.env` file. which will secure the redis running inside docker-compose with this password. This will be automatically picked up by app server and sidekiq, to authenticate while making connections to redis server.
+## Development environment
 
-```bash
-docker-compose build
-```
+1. Clone the repository.
 
-After building the image or after destroying the stack you would have to reset the database using following command
+    ```bash
+    $ git clone https://github.com/chatwoot/chatwoot.git
+    ```
 
-```bash
-docker-compose run --rm rails bundle exec rails db:reset
-```
+2. Make a copy of the example environment file and modify as required [optional].
 
-### Running the app
+    ```bash
+    $ cp .env.example .env
+    ```
 
-```bash
-docker-compose run --rm --service-port rails
```

Comment:
@vishnu-narayanan  could you mention this line at the end of this document in a section somewhere like. 

## using debuggers while working with docker. 

use the following command instead of  `docker-compose up` if you want to use debuggers like `byebug` or `binding.pry` while working on the chatwoot code

 ```bash
docker-compose run --rm --service-port rails
```

Reply:
@sojan-official sure.